### ISSUE TITLE
[Parse Error] ReferenceError: c is not defined

### ISSUE THREAD

**Issue by nevadascout** [HvyIndustries/crane#125]
Reported by @carloscz 

```
[INFO] Checking the current version of Crane
[INFO] Watching these files: {**/*.phpt,**/*.php}
[INFO] Preparing to parse 1446 PHP source files...
[INFO] Processing 0 stubs from C:\Users\Holub\AppData\Roaming/Crane/phpstubs
[INFO] Parsed file 1 of 1446 : d:\stuff\projekty\logio\PW branch\core\_check_all.php
[INFO] Parsed file 2 of 1446 : d:\stuff\projekty\logio\PW branch\core\autobackup.php
[INFO] Parsed file 3 of 1446 : d:\stuff\projekty\logio\PW branch\core\find.php
[INFO] Parsed file 4 of 1446 : d:\stuff\projekty\logio\PW branch\core\debug-mode.php
[INFO] Parsed file 5 of 1446 : d:\stuff\projekty\logio\PW branch\core\clean-svn.php
[INFO] Parsed file 6 of 1446 : d:\stuff\projekty\logio\PW branch\core\index.php
[INFO] Parsed file 7 of 1446 : d:\stuff\projekty\logio\PW branch\core\icli.php
[INFO] Parsed file 8 of 1446 : d:\stuff\projekty\logio\PW branch\core\run-batch.php
[INFO] Parsed file 9 of 1446 : d:\stuff\projekty\logio\PW branch\core\pwcp.php
[INFO] Parsed file 10 of 1446 : d:\stuff\projekty\logio\PW branch\core\run.php
[INFO] Parsed file 11 of 1446 : d:\stuff\projekty\logio\PW branch\core\sync.php
[INFO] Parsed file 12 of 1446 : d:\stuff\projekty\logio\PW branch\core\system-nedostupny.php
[INFO] Parsed file 13 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\check.php
[INFO] Parsed file 14 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\check-debug-mode.php
[INFO] Parsed file 15 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\clearcache.php
[INFO] Parsed file 16 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\data.php
[INFO] Parsed file 17 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\crashed.php
[INFO] Parsed file 18 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\cli.php
[INFO] Parsed file 19 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\dbsync-check.php
[INFO] Parsed file 20 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\dbsync-logio.php
[INFO] Parsed file 21 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\dbsync.php
[INFO] Parsed file 22 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\mp.php
[INFO] Parsed file 23 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\errorpage.php
[INFO] Parsed file 24 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\project.php
[INFO] Parsed file 25 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\run_from_cli.php
[INFO] Parsed file 26 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\run.php
[INFO] Parsed file 27 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\test-dbsync-logio.php
[INFO] Parsed file 28 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\tools.php
[INFO] Parsed file 29 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Configurator.php
[INFO] Parsed file 30 of 1446 : d:\stuff\projekty\logio\PW branch\core\tests\bootstrap.php
[INFO] Parsed file 31 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\RobotLoader.php
[INFO] Parsed file 32 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\autobackup-run.php
[INFO] Parsed file 33 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\index.php
[INFO] Parsed file 34 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\autobackup.php
[INFO] Parsed file 35 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tunel-mapper.php
[INFO] Parsed file 36 of 1446 : d:\stuff\projekty\logio\PW branch\core\c_modules\c_batch_inventory\test.php
[INFO] Parsed file 37 of 1446 : d:\stuff\projekty\logio\PW branch\core\c_modules\c_batch_inventory\test_simple.php
[INFO] Parsed file 38 of 1446 : d:\stuff\projekty\logio\PW branch\core\c_modules\c_batch_inventory\test_simple_object.php
[INFO] Parsed file 39 of 1446 : d:\stuff\projekty\logio\PW branch\core\develop\noter\index.php
[INFO] Parsed file 40 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\abstract-html-module.php
[INFO] Parsed file 41 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\abstract-module.php
[INFO] Parsed file 42 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\ModulesLatteFileLoader.php
[INFO] Parsed file 43 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\abstract-module.unit.phpt
[INFO] Parsed file 44 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\exception.php
[INFO] Parsed file 45 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\framework.php
[INFO] Parsed file 46 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\global-utilities.php
[INFO] Parsed file 47 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\settings.unit.phpt
[INFO] Parsed file 48 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\settings.php
[INFO] Parsed file 49 of 1446 : d:\stuff\projekty\logio\PW branch\core\framework\classes\modules.php
[INFO] Parsed file 50 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\access-right\access-right.php
[INFO] Parsed file 51 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin-log-screen\admin-log-screen.php
[INFO] Parsed file 52 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\adverts\adverts.php
[INFO] Parsed file 53 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\caching\caching.unit.phpt
[INFO] Parsed file 54 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\caching\caching.php
[INFO] Parsed file 55 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\configuration\configuration.php
[INFO] Parsed file 56 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\configuration\configuration.unit.phpt
[INFO] Parsed file 57 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\content\content.php
[INFO] Parsed file 58 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\db\PwPDOException.php
[INFO] Parsed file 59 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\db\PwDibiEvent.php
[INFO] Parsed file 60 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\db\PwPDOStatement.php
[INFO] Parsed file 61 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\debug-mode\debug-mode.php
[INFO] Parsed file 62 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\db\db.php
[INFO] Parsed file 63 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\error-reporter\error-reporter.php
[INFO] Parsed file 64 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\email-buffer\email-buffer.php
[INFO] Parsed file 65 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\gantt\gantt.php
[INFO] Parsed file 66 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\mailer\mailer.php
[INFO] Parsed file 67 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\lang\lang.php
[INFO] Parsed file 68 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\pluginprocessor\pluginprocessor.php
[INFO] Parsed file 69 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\raw-to-udm\raw-to-udm.php
[INFO] Parsed file 70 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\screen-process\screen-process.php
[INFO] Parsed file 71 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\smarty\smarty.php
[INFO] Parsed file 72 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\sms-sender\sms-sender.php
[INFO] Parsed file 73 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\search\search.php
[INFO] Parsed file 74 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\status\status.php
[INFO] Parsed file 75 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\tinyajax\tinyajax.php
[INFO] Parsed file 76 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\treeview\treeview.php
[INFO] Parsed file 77 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\vdaemon\config.php
[INFO] Parsed file 78 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\ConsistencyCheckerFactory.php
[INFO] Parsed file 79 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\ConsistencyChecker.php
[INFO] Parsed file 80 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\ErrorException.php
[INFO] Parsed file 81 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\IPacketSender.php
[INFO] Parsed file 82 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\Exception.php
[INFO] Parsed file 83 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\vdaemon\vdaemon.php
[INFO] Parsed file 84 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\PacketSender.php
[INFO] Parsed file 85 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\ConsistencyChecker\WarningException.php
[INFO] Parsed file 86 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Framework\DatabaseOptimizer.php
[INFO] Parsed file 87 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Logging\FatalErrorStatusReporter.php
[INFO] Parsed file 88 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Logging\Logger.php
[INFO] Parsed file 89 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Models\Entity.php
[INFO] Parsed file 90 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Models\Repository.php
[INFO] Parsed file 91 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Services\BatchKiller.php
[INFO] Parsed file 92 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Router\RouterFactory.php
[INFO] Parsed file 93 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Services\FrameworkProxy.php
[INFO] Parsed file 94 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Services\IBatchKiller.php
[INFO] Parsed file 95 of 1446 : d:\stuff\projekty\logio\PW branch\core\tests\framework\CFramework.unit.phpt
[INFO] Parsed file 96 of 1446 : d:\stuff\projekty\logio\PW branch\core\src\Services\Run.php
[INFO] Parsed file 97 of 1446 : d:\stuff\projekty\logio\PW branch\core\tests\framework\Configurator.unit.phpt
[INFO] Parsed file 98 of 1446 : d:\stuff\projekty\logio\PW branch\core\tests\framework\Run.unit.phpt
[INFO] Parsed file 99 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\api.php
[INFO] Parsed file 100 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\bootstrap.php
[INFO] Parsed file 101 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\create-send-stub.php
[INFO] Parsed file 102 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\cleanup-neptun.php
[INFO] Parsed file 103 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\download-stub.php
[INFO] Parsed file 104 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\pwdate.php
[INFO] Parsed file 105 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\tester\index.php
[ERROR] There was a problem parsing PHP file: d:\stuff\projekty\logio\PW branch\core\_db\clean-db-for-new-projects\_01_prepare_clean_version_for_update.php
[ERROR] [ReferenceError: c is not defined]
[ERROR] There was a problem parsing PHP file: d:\stuff\projekty\logio\PW branch\core\_db\clean-db-for-new-projects\_02_dump_result.php
[ERROR] [ReferenceError: c is not defined]
[ERROR] There was a problem parsing PHP file: d:\stuff\projekty\logio\PW branch\core\_db\clean-db-for-new-projects\step1.php
[ERROR] [ReferenceError: c is not defined]
[ERROR] There was a problem parsing PHP file: d:\stuff\projekty\logio\PW branch\core\_db\clean-db-for-new-projects\step2.php
[ERROR] [ReferenceError: c is not defined]
[INFO] Parsed file 110 of 1446 : d:\stuff\projekty\logio\PW branch\core\develop\scripts\pw-module-generator\index.php
[INFO] Parsed file 111 of 1446 : d:\stuff\projekty\logio\PW branch\core\c_modules\c_forecast_loader\test\t.php
[INFO] Parsed file 112 of 1446 : d:\stuff\projekty\logio\PW branch\core\c_modules\c_forecast\test\t.php
[INFO] Parsed file 113 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\abc-calculation\abc-calculation.unit.phpt
[INFO] Parsed file 114 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\abc-display\abc-display.php
[INFO] Parsed file 115 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\abc-display-custom-aggregation\abc-display-custom-aggregation.php
[INFO] Parsed file 116 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\abc-calculation\abc-calculation.php
[INFO] Parsed file 117 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\abc-histogram\abc-histogram.php
[INFO] Parsed file 118 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\abc\report-abc\report-abc.php
[INFO] Parsed file 119 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-abc\admin-abc.php
[INFO] Parsed file 120 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-back-redistribution\admin-back-redistribution.php
[INFO] Parsed file 121 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-basestock\admin-basestock.php
[INFO] Parsed file 122 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-basket\admin-basket.php
[INFO] Parsed file 123 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-basestock-max\admin-basestock-max.php
[INFO] Parsed file 124 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-batch\admin-batch.unit.phpt
[INFO] Parsed file 125 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-batch-history\admin-batch-history.php
[INFO] Parsed file 126 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-batch\admin-batch.php
[INFO] Parsed file 127 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-batch-manual\admin-batch-manual.php
[INFO] Parsed file 128 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-cashflow\admin-cashflow.php
[INFO] Parsed file 129 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-bookmark\admin-bookmark.php
[INFO] Parsed file 130 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-cashflow-v2\admin-cashflow-v2.php
[INFO] Parsed file 131 of 1446 : d:\stuff\projekty\logio\PW branch\core\tools\convert-pwm.php
[INFO] Parsed file 132 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-category-order-relation\admin-category-order-relation.php
[INFO] Parsed file 133 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-category-preferred-product-provider\admin-category-preferred-product-provider.php
[INFO] Parsed file 134 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-container\admin-container.php
[INFO] Parsed file 135 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-comfortable-sl-settings\admin-comfortable-sl-settings.php
[INFO] Parsed file 136 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-customer-service\admin-customer-service.php
[INFO] Parsed file 137 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-data-obfuscator\admin-data-obfuscator.php
[INFO] Parsed file 138 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-detail\admin-detail.php
[INFO] Parsed file 139 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-distribute-order-between-stock-and-supplier\admin-distribute-order-between-stock-and-supplier.php
[INFO] Parsed file 140 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-customer-service-reloaded\admin-customer-service-reloaded.php
[INFO] Parsed file 141 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-eoq\admin-eoq.php
[INFO] Parsed file 142 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-extreme-cleanup\admin-extreme-cleanup.php
[INFO] Parsed file 143 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-extreme-customers\admin-extreme-customers.php
[INFO] Parsed file 144 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-forecast-fast-adapt\admin-forecast-fast-adapt.php
[INFO] Parsed file 145 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-forecast-history-cutoff\admin-forecast-history-cutoff.php
[INFO] Parsed file 146 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-holidays\admin-holidays.php
[INFO] Parsed file 147 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-manual-ofc\admin-manual-ofc.php
[INFO] Parsed file 148 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-logview\admin-logview.php
[INFO] Parsed file 149 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-import\admin-import.php
[INFO] Parsed file 150 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-new-product-ordering\admin-new-product-ordering.php
[INFO] Parsed file 151 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-non-profit-items\admin-non-profit-items.php
[INFO] Parsed file 152 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-new-products-reward\admin-new-products-reward.php
[INFO] Parsed file 153 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-order-method\admin-order-method.php
[INFO] Parsed file 154 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-order-period\admin-order-period.php
[INFO] Parsed file 155 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-overwrite-abc-for-product\admin-overwrite-abc-for-product.php
[INFO] Parsed file 156 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-ordering-exception-editor\admin-ordering-exception-editor.php
[INFO] Parsed file 157 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-pcat\admin-pcat.php
[INFO] Parsed file 158 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-period-sale-interval\admin-period-sale-interval.php
[INFO] Parsed file 159 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering\admin-periodical-ordering.php
[INFO] Parsed file 160 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-clients-groups\admin-periodical-ordering-clients-groups.php
[INFO] Parsed file 161 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-configuration\admin-periodical-ordering-configuration.php
[INFO] Parsed file 162 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-custom-fields\admin-periodical-ordering-custom-fields.php
[INFO] Parsed file 163 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-products-groups\admin-periodical-ordering-products-groups.php
[INFO] Parsed file 164 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-providers\admin-periodical-ordering-providers.php
[INFO] Parsed file 165 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-predecessor-successor\admin-predecessor-successor.php
[INFO] Parsed file 166 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-redistribution\admin-periodical-ordering-redistribution.php
[INFO] Parsed file 167 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-periodical-ordering-warehouses-groups\admin-periodical-ordering-warehouses-groups.php
[INFO] Parsed file 168 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-process\admin-process.php
[INFO] Parsed file 169 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-promo\admin-promo.php
[INFO] Parsed file 170 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-administration\admin-provider-administration.php
[INFO] Parsed file 171 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider\admin-provider.php
[INFO] Parsed file 172 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-groups\admin-provider-groups.php
[INFO] Parsed file 173 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-email\admin-provider-email.php
[INFO] Parsed file 174 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-periodical-ordering\admin-provider-periodical-ordering.unit.phpt
[INFO] Parsed file 175 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-price-markup\admin-provider-price-markup.php
[INFO] Parsed file 176 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-provider-periodical-ordering\admin-provider-periodical-ordering.php
[INFO] Parsed file 177 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-redistribution\admin-redistribution.php
[INFO] Parsed file 178 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-role\admin-role.php
[INFO] Parsed file 179 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-rounding\admin-rounding.php
[INFO] Parsed file 180 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-run-batch\admin-run-batch.php
[INFO] Parsed file 181 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-safetystock\admin-safetystock.php
[INFO] Parsed file 182 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-scenario\admin-scenario.php
[INFO] Parsed file 183 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-scenario-settings\admin-scenario-settings.php
[INFO] Parsed file 184 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-seasonality\admin-seasonality.php
[INFO] Parsed file 185 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-similar-history\admin-similar-history.php
[INFO] Parsed file 186 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-similar-source\admin-similar-source.php
[INFO] Parsed file 187 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-similar-target\admin-similar-target.php
[INFO] Parsed file 188 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-smi\admin-smi.php
[INFO] Parsed file 189 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-translate\admin-translate.php
[INFO] Parsed file 190 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-user\admin-user.php
[INFO] Parsed file 191 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-user-accounts\admin-user-accounts.php
[INFO] Parsed file 192 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-user-settings\admin-user-settings.php
[INFO] Parsed file 193 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-stockout-fulfillment\admin-stockout-fulfillment.php
[INFO] Parsed file 194 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-userview\admin-userview.php
[INFO] Parsed file 195 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-userview-multi\admin-userview-multi.php
[INFO] Parsed file 196 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-viewtype\admin-viewtype.php
[INFO] Parsed file 197 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-wacc\admin-wacc.php
[INFO] Parsed file 198 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\admin\admin-warehouse-capacity\admin-warehouse-capacity.php
[INFO] Parsed file 199 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticator-basic\login-authenticator-basic.php
[INFO] Parsed file 200 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticator-external\login-authenticator-external.php
[INFO] Parsed file 201 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticator-ldap-example\login-authenticator-ldap-example.php
[INFO] Parsed file 202 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-smart\basestock-smart.php
[INFO] Parsed file 203 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-forwardsales\basestock-forwardsales.php
[INFO] Parsed file 204 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-backwardsales\basestock-backwardsales.php
[INFO] Parsed file 205 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock\basestock.php
[INFO] Parsed file 206 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-loader\basestock-loader.php
[INFO] Parsed file 207 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-zerosales\basestock-zerosales.php
[INFO] Parsed file 208 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticate\AuthenticationException.php
[INFO] Parsed file 209 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticate\ILoginAuthenticator.php
[INFO] Parsed file 210 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\basestock\basestock-v2\basestock-v2.php
[INFO] Parsed file 211 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\authenticate\login-authenticate\login-authenticate.php
[INFO] Parsed file 212 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\aggregated-day-progress\aggregated-day-progress.php
[INFO] Parsed file 213 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\abstract-report\abstract-report.php
[INFO] Parsed file 214 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\breakup-analyse\breakup-analyse.php
[INFO] Parsed file 215 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\breakup-display\breakup-display.php
[INFO] Parsed file 216 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\cleanup-leadtime\cleanup-leadtime.php
[INFO] Parsed file 217 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\cleanup-sale\cleanup-sale.php
[INFO] Parsed file 218 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\day-progress\day-progress.php
[INFO] Parsed file 219 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\day-progress-central\day-progress-central.php
[INFO] Parsed file 220 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\day-progress-future\day-progress-future.php
[INFO] Parsed file 221 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\economic-order-quantity-display\economic-order-quantity-display.php
[INFO] Parsed file 222 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\ewmatrix\ewmatrix.php
[INFO] Parsed file 223 of 1446 : d:\stuff\projekty\logio\PW branch\core\modules\analyse\first-delivery-new-store-suggestion\first-delivery-new-store-suggestion.php
```

**Comment by nevadascout** [comment]
Is it possible to get a copy of the files that are throwing the `ReferenceError: c is not defined` error?

**Comment by nevadascout** [comment]
The cause of this has been found and will be resolved with #151
